### PR TITLE
added markdown support for blogs

### PR DESCRIPTION
**Pull request by Abhishek-More**
## Summary

We want to use Github as a CMS for blogs. Added ability to read from markdown files in blog-content/

### CODE REVIEW

File: highlight.io/pages/blog/[slug].tsx [hunk 1]
```diff
@@ -237,11 +317,11 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
 		})
 	}
 
-	if (!data.post.author?.profilePhoto?.url) {
-		throw new Error(
-			`missing required profile image for blog '${data.post.slug}', author: ${data.post.author?.profilePhoto?.url}.`,
-		)
-	}
+	// if (!data.post.author?.profilePhoto?.url) {
```
Comment (jay-khatri) [inline highlight.io/pages/blog/[slug].tsx:320]: commented?

File: highlight.io/pages/blog/[slug].tsx [hunk 2]
```diff
@@ -287,17 +367,17 @@ const PostSection = ({ p }: { p: PostSection; idx: number }) => {
...
-							<Image
-								className={styles.blogImage}
-								src={props.src || ''}
-								alt={props.altText}
-								width={props.width}
-								height={props.height}
-							/>
-						</div>
-					),
+					// img: (props) => (
```
Comment (jay-khatri) [inline highlight.io/pages/blog/[slug].tsx:370]: commented?

**Review by jay-khatri — APPROVED** [review]

File: highlight.io/pages/blog/[slug].tsx (hunk 1 above)
Comment (Abhishek-More) [reply highlight.io/pages/blog/[slug].tsx]: Removed comments

File: highlight.io/pages/blog/[slug].tsx (hunk 2 above)
Comment (Abhishek-More) [reply highlight.io/pages/blog/[slug].tsx]: Removed comments

File: blog-content/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+- names of files can only have letters and dashes
+
+TODO:
+- featured shouldn't be required
+- slug shouldn't be required
```
Comment (Vadman97) [inline blog-content/README.md:5]: is this still accurate?

**Review by Vadman97 — APPROVED** [review]
does `sitemap.ts` know to show the `blog-content` paths in the output?

**Comment by Vadman97** [comment]
@Abhishek-More looks good, feel free to merge this once the sitemap question is addressed

File: blog-content/README.md (hunk 3 above)
Comment (Abhishek-More) [reply blog-content/README.md]: Featured defaults to false, while the slug is autogenerated based on the filename. Updated the sitemap to contain the new blog-content paths.

**Pull request merged**